src/manager.cpp

```diff
@@ -337,11 +337,22 @@ static std::map<pid_t, std::string> start_modules(Config& config, MQTTAbstractio
                     fmt::format((mod.second.ready) ? TERMINAL_STYLE_OK : TERMINAL_STYLE_ERROR, "ready");
                 EVLOG_debug << fmt::format("  {}: {}", mod.first, text_ready);
             }
+            if (!standalone_modules.empty() && std::find(standalone_modules.begin(), standalone_modules.end(),
+                                                         module_name) != standalone_modules.end()) {
+                EVLOG_info << fmt::format("Standalone module {} initialized.", module_name);
+            }
             if (std::all_of(modules_ready.begin(), modules_ready.end(),
                             [](const auto& element) { return element.second.ready; })) {
                 EVLOG_info << fmt::format(TERMINAL_STYLE_OK,
                                           ">>> All modules are initialized. EVerest up and running <<<");
                 mqtt_abstraction.publish("everest/ready", nlohmann::json(true));
+            } else if (!standalone_modules.empty()) {
+                auto modules_spawned = std::count_if(modules_ready.begin(), modules_ready.end(),
+                                                     [](const auto& element) { return element.second.ready; });
+                if (modules_spawned == modules_ready.size() - standalone_modules.size()) {
```

Comment:
The code is not completely correct, because it could also happen, that a standalone module starts before one of the automatically ones and then the message would be incorrect, but I doubt this will ever happen or have any impact.

Reply:
The manager probably would not even realize that such a standalone module was started early because it needs to have the module ready topic subscribed before a standalone module can start. But yes, there is a small race condition here that most likely will not matter in the usecase for this particular functionality (starting a module by-hand for debugging or testing purposes)

Reply:
:+1: